.github/workflows/ci.yml

```diff
@@ -0,0 +1,157 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches:
+    tags:
+  pull_request:
+
+jobs:
+  coding-standard:
+    runs-on: ubuntu-18.04
+    name: Coding Standard
+
+    steps:
+      -   uses: actions/checkout@v2
+
+      -   name: Install PHP
+          uses: shivammathur/setup-php@1.7.0
+          with:
+            php-version: 7.3
+            coverage: none
+            extensions: json, mbstring
+
+      -   name: Get Composer Cache Directory
+          id: composer-cache
+          run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      -   name: Cache dependencies
+          uses: actions/cache@v1
+          with:
+            path: ${{ steps.composer-cache.outputs.dir }}
+            key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
+            restore-keys: ${{ runner.os }}-composer-
+
+      -   name: Install Dependencies
+          run: composer install ${DEPENDENCIES}
+
+      -   name: Coding Standard
+          run: vendor/bin/phpcs
```

Comment:
před CS bys měl pouštět linter

Reply:
jakej linter? a proc?

Reply:
https://github.com/JakubOnderka/PHP-Parallel-Lint , používáme ho všude, nevím proč seš tak překvapenej :D 

Důvod je simple, je to nejrychlejší check co kontroluje největší hrubky ... a většina CS sniffů se rozbije, když kód není validní

Reply:
to je zbytecnej step. rozbije se to tak jako tak

Reply:
Není to zbytečnej step, proč myslíš že to existuje? :D Není záruka že se to někde rozbije

Reply:
phpcs, phpstan a testy ti urcite neprojdou kdyz ten kod nebude validni. AFAIK to je z historickejch duvodu, jinak nevim k cemu by to bylo 🤔

Reply:
Pokud máš jako podmínku buildu 100% coverage, tak jo

Reply:
udelej si syntax error a zkus aby ti prosel phpstan. typek to neda

Reply:
Ok, tak je cool že to spadne ... jen to spadne na nějakým debilním erroru no, lint ti vyplivne co přesně je kde špatně no ... přijde mi že není bad to pouštět, je to přecijen skoro instant ...

Reply:
Kdyžtak probereme v po s ostatníma

Reply:
je to dalsi step navic, tady ho nepotrebuju

muzeme to klidne vyhodit pak i u nas ni. kdyz failne ten, tak ti stejne failne vsechno ostatni proste :D hlavne v ty paralerni pipeline to IMO nedava smysl no